flink/v1.18/flink/src/main/java/org/apache/iceberg/flink/FlinkSchemaUtil.java

```diff
@@ -68,6 +72,33 @@ public static Schema convert(TableSchema schema) {
     return freshIdentifierFieldIds(iSchema, schema);
   }
 
+  /** Convert the flink table schema to apache iceberg schema with column comment. */
+  public static Schema convert(ResolvedCatalogTable catalogTable) {
+    List<Column> tableColumns = catalogTable.getResolvedSchema().getColumns();
+    // copy from org.apache.flink.table.api.Schema#toRowDataType
```

Comment:
can we use `ResolvedSchema#toSourceRowDataType()` ?

Reply:
`toSourceRowDataType` cannot obtain the comment information of the columns